zone/groups.cpp

```diff
@@ -1126,7 +1140,13 @@ void Group::TeleportGroup(Mob* sender, uint32 zoneID, uint16 instance_id, float
 }
 
 bool Group::LearnMembers() {
-	std::string query = StringFormat("SELECT name FROM group_id WHERE groupid = %lu", (unsigned long)GetID());
+	//std::string query = StringFormat("SELECT name FROM group_id WHERE groupid = %lu", (unsigned long)GetID());
+	std::string query = StringFormat("SELECT name FROM group_id "
+		"WHERE group_id.groupid = %lu AND group_id.name NOT "
+		"IN(SELECT group_leaders.leadername FROM group_leaders WHERE gid = %lu)"
+		, (unsigned long)GetID()
+		, (unsigned long)GetID());
+
```

Comment:
Formatting, also don't have to do `unsigned long` as `GetID()` is already a `uint32` and the string formatting doesn't really mind it.

Reply:
Updated.